lib/active_merchant/billing/gateways/paypal_checkout.rb

```diff
@@ -0,0 +1,376 @@
+require 'active_merchant/billing/gateways/paypal/paypal_checkout_common'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PaypalCheckoutGateway < Gateway
+      include PaypalCheckoutCommon
+      
+      self.supported_countries = ['AU', 'AT', 'BE', 'BG', 'CA', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'GR', 'HU', 'IT', 'LV', 'LI', 'LT', 'LU', 'MT', 'NL', 'NO', 'PL', 'PT', 'RO', 'SK', 'SI', 'ES', 'SE', 'US', 'GB']
+      self.homepage_url        = 'https://www.paypal.com/us/business/platforms-and-marketplaces'
+      self.display_name        = 'PayPal Commerce Platform'
+      self.default_currency    = 'USD'
+      self.supported_cardtypes = [:visa, :master, :american_express, :discover, :jcb]

```

Comment:
https://github.com/inaitech/active_merchant/pull/5/files#diff-c18b98315e774d346fa9a911a9ff4b6f62b92461809dea61532a6c85021b552cR12
This seems to have bigger list. what's the difference?

Reply:
This list is used for verifying the card type that is provided but we are not using it anywhere. I'll list all the card types here in case we are to use in future.